README.md

```diff
@@ -1001,8 +1001,6 @@ DeviceInfo.getTotalMemory().then((totalMemory) => {
 
 ### getUniqueId()
 
-This is a constant and may be referenced directly
-
```

Comment:
Maybe "This is considered sensitive information in some app stores (for example Huawei). For those stores you must obtain user permission prior to accessing it" ?